test/NuGet.Core.Tests/NuGet.Configuration.Test/EnvironmentSupportTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using NuGet.Test.Utility;
+using System;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+namespace NuGet.Configuration.Test
+{
+    public class EnvironmentSupportTests
+    {
+        string DefaultNuGetConfigurationWithEnvironmentVariable = @"
+<configuration>
+    <config>
+        <add key='repositoryPath' value='%RP_ENV_VAR%\two' />
+    </config>
+</configuration>";
+
+        [Fact]
+        public void GetValueEvalutatesEnvironmentVariable()
+        {
+            //Arrange
+            var expectedRepositoryPath = @"ONE\two";
+
+            using (var nugetConfigFileFolder = TestFileSystemUtility.CreateRandomTestFolder())
+            {
+                var nugetConfigFile = "NuGetDefaults.config";
```

Comment:
I would change the name of NuGetDefaults.config to be nuget.config to avoid confusion about the intention of the test. The product change impacts all settings. NuGetDefaults.config is a special nuget config file intended for ConfigurationDefaults